### ISSUE TITLE
Add support for `GetActiveDeviceInfo.cgi`

### ISSUE THREAD

**Issue by farmio** [nielstron/pyfronius#32]
This could be used to discovery all devices connected to a known Datalogger (when using `DeviceClass=System`). 
```jsonc
curl http://10.1.2.5/solar_api/v1/GetActiveDeviceInfo.cgi\?DeviceClass\=System
{
   "Body" : {
      "Data" : {
         "Inverter" : {
            "1" : {
               "DT" : 122,
               "Serial" : "30412345"
            }
         },
         "Meter" : {
            "0" : {
               "DT" : -1,
               "Serial" : "18412345"
            }
         },
         "Ohmpilot" : {},
         "SensorCard" : {},
         "Storage" : {},
         "StringControl" : {}
      }
   },
   "Head" : {
      "RequestArguments" : {
         "DeviceClass" : "System"
      },
      "Status" : {
         "Code" : 0,
         "Reason" : "",
         "UserMessage" : ""
      },
      "Timestamp" : "2021-08-17T14:12:17+02:00"
   }
}
```
Currently possible devices are:
- "Inverter" 
- "Storage" 
- "Ohmpilot"
- "SensorCard" 
- "StringControl" 
- "Meter"

**Comment by nielstron** [comment]
Feel free to test branch `active_device_info`. If it works, I can merge it into `dev`. Anyways I added tests based on your data, so it should be fine.

**Comment by farmio** [comment]
Cool! Works great on my system. Output for 
```python
# ...
res = await fronius.current_active_device_info()
print(res)
# ...
```
is (new-lines added)
```python
DEBUG:pyfronius:using highest supported API version API_VERSION.V1
DEBUG:pyfronius:Get current active device info data for GetActiveDeviceInfo.cgi?DeviceClass=System
DEBUG:pyfronius:Converting system active device data: '{'Inverter': {'1': {'DT': 122, 'Serial': '30412345'}}, 'Meter': {'0': {'DT': -1, 'Serial': '18412345'}}, 'Ohmpilot': {}, 'SensorCard': {}, 'Storage': {}, 'StringControl': {}}'
{'timestamp': {'value': '2021-08-17T20:33:07+02:00'}, 
'status': {'Code': 0, 'Reason': '', 'UserMessage': ''}, 
'inverters': [{'device_id': '1', 'device_type': 122, 'serial_number': '30412345'}], 
'meters': [{'device_id': '0', 'serial_number': '18412345'}], 
'ohmpilots': [], 
'sensor_cards': [], 
'storages': [], 
'string_controls': []}
```

Just one little thing:
"Ohmpilot" is supported since version 3.6.1-3, but "DeviceClass=System" since version 3.3.4-5. So there is a small gap between this which makes me think https://github.com/nielstron/pyfronius/blob/e0978fcdfc1b1f8a647ab2d4b40aae3714ea8377/pyfronius/__init__.py#L908 should be guarded so it can't raise `KeyError` on these versions.

**Comment by nielstron** [comment]
Good point, the access is now guarded and no information returned in case it is not given by the overview. Note: When automatically determining the list of installed devices, we need to keep in mind that additional devices might be present but are not shown due to the firmware version.

**Comment by nielstron** [comment]
WIll be included in 0.6.0
